server/pytest.ini

```diff
@@ -2,15 +2,9 @@
 DJANGO_SETTINGS_MODULE = portal.settings.unit_test_settings
 python_files = tests.py *unit_test.py
 
-filterwarnings =
-    ignore::DeprecationWarning:django.*:
-    ignore::DeprecationWarning:openapi_schema_validator.*:
-    ignore::DeprecationWarning:openapi_spec_validator.*:
-    ignore::DeprecationWarning:openapi_core.*:
-    ignore::DeprecationWarning:kombu.*:
-    ignore::DeprecationWarning:pkg_resources.*:
-    ignore:.*Django now detects this configuration.*:django.utils.deprecation.RemovedInDjango41Warning
-
+filterwarnings = 
+    ignore::DeprecationWarning:twisted.*:
+    ignore::django.utils.deprecation.RemovedInDjango50Warning
 markers =
```

Comment:
Still need to filter a couple of warnings. 

- The `twisted` package has a [PR](https://github.com/twisted/twisted/issues/11848) open currently to address the cgi deprecation warning 
- We can ignore any RemovedInDjango50Warning since Django 5 support is still limited at the moment